runner/src/main.rs

```diff
@@ -319,26 +351,42 @@ fn run_example(example: &Example) -> Step {
                     "./bazel-bin/examples/{}/config/config.bin",
                     example.name
                 )),
-                foreground: Box::new(Step::Single {
+                foreground: Box::new(Step::Multiple {
                     name: "run_client".to_string(),
-                    command: Cmd::new(
-                        "bazel",
-                        vec![
-                            "run".to_string(),
-                            "--".to_string(),
-                            format!("//examples/{}/client:client", example.name),
-                            "--ca_cert=../../../../../../../../examples/certs/local/ca.pem"
-                                .to_string(),
-                        ]
+                    steps: example
+                        .clients
                         .iter()
-                        .chain(example.additional_client_flags.iter()),
-                    ),
+                        .map(|(name, client)| run_client(name, &client))
+                        .collect(),
```

Comment:
That's nice, it seems like this PR is unofficially adding support for running multiple clients